PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/storage/azurite/actions/StartAzuriteAction.kt

```diff
@@ -108,6 +112,15 @@ class StartAzuriteAction
                     "--location",
                     azuriteWorkspaceLocation)
 
+            if (supportsTableStorage(nodeJsLocalInterpreter.interpreterSystemDependentPath, azuriteJsFile.absolutePath)) {
```

Comment:
What do you think about adding a bit more logging here? E.g. log `supportTableStorage` results? And store some tracing inside a method itself.